### ISSUE TITLE
using scene-level labels does increase performance?

### ISSUE THREAD

**Issue by cervantes-loves-ai** [ekosman/AnomalyDetectionCVPR2018-Pytorch#83]
How about comparing with the proposed method itself using scene-level labels (by removing the MIL layer). I assume the performance is better than using group-level labels, but what is the gap between them?

This method have multiple components, e.g. feature extractor, loss function, MIL layer, which component contributes to the learning capacity given limited labeled samples?

thanks in advance.

**Comment by ekosman** [comment]
I have plans to add more experiments in the future (e.g. use different feature extractors). I'll do that as soon as I have more time for this.
